Mage/src/main/java/mage/abilities/effects/common/CopyEffect.java

```diff
@@ -136,8 +136,14 @@ protected boolean copyToPermanent(Permanent permanent, Game game, Ability source
         permanent.setStartingDefense(copyFromObject.getStartingDefense());
         if (copyFromObject instanceof Permanent) {
             Permanent targetPermanent = (Permanent) copyFromObject;
-            permanent.setTransformed(targetPermanent.isTransformed());
-            permanent.setSecondCardFace(targetPermanent.getSecondCardFace());
+            //707.2. When copying an object, the copy acquires the copiable values of the original object’s characteristics [..]
+            //110.5. A permanent's status is its physical state. There are four status categories, each of which has two possible values:
+            // tapped/untapped, flipped/unflipped, face up/face down, and phased in/phased out.
+            // Each permanent always has one of these values for each of these categories.
+            //110.5a Status is not a characteristic, though it may affect a permanent’s characteristics.
+            //Being transformed is not a copiable characteristic, nor is the back side of a DFC
+            //permanent.setTransformed(targetPermanent.isTransformed());
+            //permanent.setSecondCardFace(targetPermanent.getSecondCardFace());
             permanent.setFlipCard(targetPermanent.isFlipCard());
```

Comment:
Is same ignore logic must be for flip fields too?

Reply:
AFAIK, the status of being flipped is a status and isn't copied but the existence of a flipped version of a card is copiable- this means that copying a flipped card does not result in a flipped clone (Unless the clone was already flipped by something else). This differs from DFCs where the current face is the only thing that is copied.

Compare
> 712.8d While a double-faced permanent has its front face up, it has only the characteristics of its front face.
> 712.8e While a transforming double-faced permanent has its back face up, it has only the characteristics of its back face. [..]

(and the similar rules for other types of DFCs) as compared to flip cards'

> 710.1b The bottom half of a flip card contains an alternative name, text box, type line, power, and toughness. These characteristics are used only if the permanent is on the battlefield and only if the permanent is flipped.

Notice how the flip card specifies that the characteristics are "used" when flipped whereas the DFCs have "only" the characteristics of their corresponding face.